handlers/event.js

```diff
@@ -82,7 +89,12 @@ module.exports.get = async (event, ctx, callback) => {
 module.exports.update = async (event, ctx, callback) => {
 
   const data = JSON.parse(event.body);
-  const id = event.queryStringParameters.id;
+
+  if (!data.hasOwnProperty('id')) {
```

Comment:
Why the switch to using `id` from the request body instead of query params?

Reply:
It doesn't seem necessary that you need to provide the query param when you need to include stuff in the body anyways, especially since Serverless doesn't enforce the query param being mandatory for the call

Reply:
But we can enforce the query param ourselves if we want to?
If we are removing the query param here, we should remove it for all functions I think?

Reply:
We are reinforcing the ID in data, so there's no point in having ID in both data and in the query params. I removed the query params from update functions only, since update functions contain a body with data. The query params are kept for get functions that don't have a body, and these get functions also check that there is an ID given

Reply:
Sounds good to me